document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/kafka/configurationHelper/KafkaConsumerProducerConfigHelper.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.document.generator.core.kafka.configurationHelper;
+
+
+import uk.gov.companieshouse.kafka.consumer.ConsumerConfig;
+import uk.gov.companieshouse.kafka.producer.ProducerConfig;
+
+public interface KafkaConsumerProducerConfigHelper {
```

Comment:
Would "KafkaBrokerConfigHelper" be more relevant to the behaviour of the interface (is it likely to contain more than these methods)? KafkaConfigHelper and KafkaConsumerProducerConfigHelper sound similar.